app/html-components/Table.tsx

```diff
@@ -16,6 +16,18 @@ const Table = styled.table<{ variant?: string; readOnly?: boolean }>`
     & th {
       min-width: ${(props) => (props.variant === 'mini' ? '30px' : '140px')};
     }
+
+    & th[class*='align-first-header'] {
```

Comment:
1. there is a easy way to select elements by `class` names
2. let's drop `header` make it simply but descriptive class names
3. are we trying to center the headers?